tools/cockpit.spec

```diff
@@ -78,6 +78,7 @@ BuildRequires: glib2-devel >= 2.37.4
 BuildRequires: systemd-devel
 BuildRequires: polkit
 BuildRequires: pcp-libs-devel
+BuildRequires: gdb
```

Comment:
The only problem with that, is that it will not help with crashes that happen in Brew or Koji. And this process has been crashing there. Having gdb around during build allows us to just use it in cases like this. The build requirements are almost always automatically pulled down, so it doesn't seem like such a problem.